qemu/tests/ksm_overcommit.py

```diff
@@ -506,9 +508,13 @@ def split_parallel():
         # mem: Memory of the guest systems. Maximum must be less than
         # host's physical ram
         mem = int(overcommit * host_mem / vmsc)
+        # If guest memory is too much, it will take very very long time for ksm
+        # to scan and merge same pages, sometimes will hit timeout
+        if mem > 8192:
+            mem = 8192
```

Comment:
With this change, no overcommit if host mem is large, but just kvm function be tested.  I wonder if it is enough for the test scenario.

Reply:
Yes, when host memory is large, there might not be overcommit happens. But still ksm should work. For large host, ksm would take much much more time to scan and merge pages, even 2 days are not long enough in my test. So I just readjust guest memory. I think the ksm function is the most important thing we care.